src/controllers/update/update.filing.date.controller.ts

```diff
@@ -21,6 +21,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     return res.render(config.UPDATE_FILING_DATE_PAGE, {
       backLinkUrl: config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
       templateName: config.UPDATE_FILING_DATE_PAGE,
+      chsUrl: process.env.CHS_URL,
```

Comment:
the CHS_URL is already in the config file src/config/index.ts so you should be able to import it and use 
chsUrl: config.CHS_URL